### DISCUSSION TITLE
不支持的 16 位应用程序

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#2188] [提问] [answered]
如题，如图
![~7S`3SW2V%}`~ 4QM7NB574](https://github.com/Hex-Dragon/PCL2/assets/109946426/c675a2d5-8c1a-4388-b64f-ba368696d482)

**Comment by MoYuan-CN** [comment]
我服务器群里的群友说有这个问题，帮忙解决下

**Comment by Chiloven945** [comment]
啊？16位？win98是吧（

**Reply by wuliaodexiaoluo** [reply]
98 是 32 位的
DOS 才是 16 位

**Reply by Ad-closeNN** [reply]
> EDIT:原来这是去年开的讨论

纠正一下，98并不算是真正意义上的32位，只是”引入了部分[32位](https://zh.wikipedia.org/wiki/32%E4%BD%8D%E5%85%83)操作系统的特性，具有一定的32位的处理能力”

[Wikipedia - Microsoft Windows](https://zh.wikipedia.org/wiki/Microsoft_Windows#:~:text=Windows%209x%E4%BB%8D%E7%84%B6%E9%9C%80%E8%A6%81%E4%BE%9D%E8%B5%9616%E4%BD%8D%E7%9A%84DOS%E5%9F%BA%E5%B1%82%E7%A8%8B%E5%BA%8F%E6%89%8D%E8%83%BD%E8%BF%90%E8%A1%8C%EF%BC%8C%E4%B8%8D%E7%AE%97%E6%98%AF%E7%9C%9F%E6%AD%A3%E6%84%8F%E4%B9%89%E4%B8%8A%E7%9A%8432%E4%BD%8D%E6%93%8D%E4%BD%9C%E7%B3%BB%E7%BB%9F)

Windows 9x系列
主条目：[Windows 9x](https://zh.wikipedia.org/wiki/Windows_9x)、[Windows 95](https://zh.wikipedia.org/wiki/Windows_95)、[Windows 98](https://zh.wikipedia.org/wiki/Windows_98)和[Windows Me](https://zh.wikipedia.org/wiki/Windows_Me)
[Windows 9x](https://zh.wikipedia.org/wiki/Windows_9x)是[Windows 95](https://zh.wikipedia.org/wiki/Windows_95)、[Windows 98](https://zh.wikipedia.org/wiki/Windows_98)、[Windows Me](https://zh.wikipedia.org/wiki/Windows_ME)等以Windows 95内核作为参考的微软操作系统通称，与[Windows NT](https://zh.wikipedia.org/wiki/Windows_NT)分离于两个开发路线。它是一种多任务图形方式的操作系统。

Windows 9x仍然需要依赖16位的DOS基层程序才能运行，不算是真正意义上的32位操作系统，由于使用DOS代码，架构也与16位DOS一样，核心属于单核心，但也引入了部分[32位](https://zh.wikipedia.org/wiki/32%E4%BD%8D%E5%85%83)操作系统的特性，具有一定的32位的处理能力。Windows 9x可视为微软将MS-DOS操作系统与早期Windows图形用户界面集成出售。

**Comment by Pigeon0v0** [comment, marked as answer]
重装系统

**Comment by NebulaTechs** [comment]
win7有十六位吗🤔

**Reply by MoYuan-CN** [reply]
我也不懂咋回事,群里一位朋友的

**Reply by NebulaTechs** [reply]
你可以告诉他可以用DosBox运行16位程序

**Comment by MoYuan-CN** [comment]
草 怎么有人挖坟

**Reply by Ad-closeNN** [reply]
草 怎么也有人来挖坟

**Comment by mc-124** [comment]
重装个屁 就是PCL2.exe损坏了
~~我也来挖坟~~
